.github/workflows/python-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: Github Actions Python Standard
+
+on: [push]
+# on:
+#   push:
+#     branches:
+#       - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
```

Comment:
might be a good time to consider bumping up the version

Reply:
Sure, I'll chat with @drazisil-codecov about the version bump as well, thanks for bringint it up @thomasrockhu 👍

Reply:
yeah ofc! let me know if I can help

Reply:
Thanks! And for sure

Reply:
👍 to the version bump.

Reply:
Are we looking at Python 3.9? @thomasrockhu @drazisil-codecov

Reply:
SGTM